packages/bpk-component-modal/src/BpkModalInner.tsx

```diff
@@ -91,14 +104,14 @@ const BpkModalInner = (props: Props) => {
       transitionTimeout={300}
     >
       <section
-        id={props.id}
-        tabIndex="-1"
+        id={rest.id}
+        tabIndex={-1}
         role="dialog"
-        aria-labelledby={props.showHeader ? headingId : null}
+        aria-labelledby={showHeader ? headingId : undefined}
         className={classNames.join(' ')}
-        ref={props.dialogRef}
+        ref={rest.dialogRef}
```

Comment:
Do we not need to define this above instead of pulling it out of `rest`? Or is this how required props are supposed to work?